### PR TITLE
Add discovery - work in progress

### PR DESCRIPTION
**Pull request by zkayyali812**
Signed-off-by: Zachary Kayyali <zkayyali@redhat.com>

open-cluster-management/backlog#9777

This PR has same changes as - https://github.com/open-cluster-management/website/pull/39 but commit has been signed off.

### CODE REVIEW

**Comment by zkayyali812** [comment]
/assign @qiujian16
Can you please review when you get a moment?

Fixed 1 noted issue in last closed [PR](https://github.com/open-cluster-management/website/pull/39) and comment has been added [here](https://github.com/open-cluster-management/website/pull/39#discussion_r591741655) regarding the other note

apologies for the second PR, I didnt sign off a commit and wasn't able to resolve it cleanly quicky

File: content/concepts/architecture.md [hunk 1]
```diff
@@ -45,3 +45,7 @@ The _Application console_ defines the user interface used to create and manage a
 ## Governance and risk (work in progress)
 
 _Governance and risk_ is the term used to define the processes that are used to manage security and compliance from the hub cluster. Ensure the security of your cluster with the extensible policy framework. After you configure a hub cluster and a managed cluster, you can create, modify and delete policies on the hub and apply policies to managed clusters.
+
+## Discovery (work in progress)
+
+_Discovery_ defines the processes that are used to identify related clusters which are available for import from Openshift Cluster Manager. By aggregating cluster information, _Discovery_ enables the user to quickly get started importing clusters to the hub. An unmanaged cluster available for import is defined as a DiscoveredCluster. A DiscoveryConfig resource must be created to enable _Discovery_. The DiscoveryConfig contains the information and filters necessary to identify DiscoveredClusters. A DiscoveredCluster resource can be imported into the _hub cluster_ at which point it becomes a ManagedCluster. For more information see the [Discovery repo](https://github.com/open-cluster-management/discovery).
```
Comment (qiujian16) [inline content/concepts/architecture.md:51]: I think this should put as a subsection of cluster lifecycle. I would recommend that "Openshift Cluster Manager" has a link.

File: content/concepts/architecture.md [hunk 2]
```diff
@@ -37,11 +37,15 @@ A multi-cluster application allows you to deploy resources on multiple clusters,
 
 The _cluster lifecycle_ defines the process the lifecycle of a cluster on the hub side. It includes the create, destroy of cluster through [hive](https://github.com/openshift/hive), the attach and detach functionality, the addon lifecycle and a UI to give a better user experience on that process.
 
+### Discovery (work in progress)
+
+_Discovery_ defines the processes that are used to identify related clusters which are available for import from [Openshift Cluster Manager](https://cloud.redhat.com/openshift/). By aggregating cluster information, _Discovery_ enables the user to quickly get started importing clusters to the hub. An unmanaged cluster available for import is defined as a DiscoveredCluster. A DiscoveryConfig resource must be created to enable _Discovery_. The DiscoveryConfig contains the information and filters necessary to identify DiscoveredClusters. A DiscoveredCluster resource can be imported into the _hub cluster_ at which point it becomes a ManagedCluster. For more information see the [Discovery repo](https://github.com/open-cluster-management/discovery).
+
 ### Application console (work in progress)
```
Comment (qiujian16) [inline content/concepts/architecture.md:44]: Not quite related, but I think Application console should under application lifecycle section?

**Review by qiujian16 — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**